### PR TITLE
finished scheduling appointment with step by step

### PR DESCRIPTION
**Pull request by JanosevicRa177**

### CODE REVIEW

**Comment by JanosevicRa177** [comment]
image

**Comment by JanosevicRa177** [comment]
image

File: src/app/patient/appointment/step-by-step/step-by-step.component.ts [hunk 1]
```diff
@@ -66,8 +82,28 @@ export class StepByStepComponent implements OnInit {
...
+        return date.toString().substring(11,16);
+    }
+    scheduleAppointment(){
+        var appointment = <ICreateAppointmentForPatient>{};
+        appointment.chosenTimeInterval = this.chosenTimeInterval;
+        appointment.doctorUid = this.chosenDoctor.uid;
+        console.log(JSON.stringify(appointment))
+        this.appointmentService.CreateAppointment(appointment).subscribe(res =>{
+            console.log(res);
+            console.log("scheduled!")
```
Comment (mihajlo182savic) [inline src/app/patient/appointment/step-by-step/step-by-step.component.ts]: Delete console log

File: src/app/patient/appointment/step-by-step/step-by-step.component.ts [hunk 2]
```diff
@@ -66,8 +82,28 @@ export class StepByStepComponent implements OnInit {
...
+        },(error) => {
+            this.cantScheduleByRoom = true;
+            console.log(error.Message)
```
Comment (mihajlo182savic) [inline src/app/patient/appointment/step-by-step/step-by-step.component.ts:101]: Delete console log

File: src/app/patient/appointment/step-by-step/step-by-step.component.ts [hunk 3]
```diff
@@ -66,8 +82,28 @@ export class StepByStepComponent implements OnInit {
...
+        this.cantScheduleByRoom = false;
+    }
+    getTimeString(date:Date): string {
+        return date.toString().substring(11,16);
+    }
+    scheduleAppointment(){
+        var appointment = <ICreateAppointmentForPatient>{};
+        appointment.chosenTimeInterval = this.chosenTimeInterval;
+        appointment.doctorUid = this.chosenDoctor.uid;
+        console.log(JSON.stringify(appointment))
```
Comment (mihajlo182savic) [inline src/app/patient/appointment/step-by-step/step-by-step.component.ts]: Delete console log

**Review by mihajlo182savic — CHANGES_REQUESTED** [review]

File: src/app/patient/appointment/step-by-step/step-by-step.component.ts (hunk 2 above)
Comment (JanosevicRa177) [reply src/app/patient/appointment/step-by-step/step-by-step.component.ts:101]: this console log should be here

**Review by mihajlo182savic — APPROVED** [review]

**Pull request merged**